src/components/TextField/TextField.tsx

```diff
@@ -260,7 +260,8 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
             <Clickable
               className={cx(styles.iconContainer, {
                 [styles.iconContainerHasIcon]: hasIcon,
-                [styles.iconContainerActive]: isPrimary
+                [styles.iconContainerActive]: isPrimary,
+                [styles.iconContainerClickable]: onIconClick !== NOOP || clearOnIconClick
```

Comment:
Nice! Let's only maybe extract it to some `const isIconContainerClickable = onIconClick !== NOOP || clearOnIconClick`, not to duplicate it twice

Reply:
Completely agree